### PR TITLE
Consume the latest kubevirt-ci fixing the kind lane.

### PR DESCRIPTION
**Pull request by fedepaol**
**What this PR does / why we need it**:

Pin the sriov config daemon version and move all the pfs into the
node's namespace.
References in https://github.com/kubevirt/kubevirtci/pull/248

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes # None

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh [hunk 1]
```diff
@@ -117,4 +121,4 @@ deploy_network_resource_injector
 sleep 5
 wait_pods_ready
 
-${SRIOV_NODE_CMD} chmod 666 /dev/vfio/vfio
+${SRIOV_NODE_CMD} chmod 666 /dev/vfio/vfio
```
Comment (phoracek) [inline cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: grrr

**Comment by phoracek** [comment]
/lgtm

For what it's worth

File: cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh (hunk 1 above)
Comment (fedepaol) [reply cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh]: :(

**Comment by fedepaol** [comment]
Relevant CI run : https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3026/pull-kubevirt-e2e-kind-k8s-sriov-1.14.2/1221780824106471424

**Comment by booxter** [comment]
For what it's worth, the SR-IOV run is green.

**Pull request merged**